lib/gollum/app.rb

```diff
@@ -111,7 +116,37 @@ class App < Sinatra::Base
     end
 
     namespace '/gollum' do
-      
+      get '/feed/' do
+        url = env['REQUEST_URI'] || "http#{env['HTTPS'] == 'off' ? nil : 's'}://#{env['SERVER_NAME']}#{env['PATH_INFO']}" # Annoyingly, env['REQUEST_URI'] is not available in the tests. In case it sometimes proves to be unavailable in production, fall back to joining together the various parts of the url.
+        url = url.match(/^(.*)gollum\/feed\/$/)[1]
+        latest_changes = "#{url}gollum/latest_changes"
+        changes = wiki_new.latest_changes(::Gollum::Page.log_pagination_options(per_page: 10, page_num: 0))
```

Comment:
Now using a hard coded maximum of 10 commits. Could also use `settings.wiki_options.fetch(:pagination_count, 10)` as for the latest changes view.

Reply:
I'm not a fan of magic numbers, so the fetching `:pagination_count` has my vote.